server/lib/Room.js

```diff
@@ -63,6 +63,88 @@ const ROUTER_SCALE_SIZE = config.routerScaleSize || 40;
 
 class Room extends EventEmitter
 {
+
+	static getLeastLoadedRouter(mediasoupWorkers, peers, mediasoupRouters)
+	{
+
+		const routerLoads = new Map();
+
+		const workerLoads = new Map();
+
+		for (const peer of peers.values())
+		{
+			const routerId = peer._routerId;
```

Comment:
Since the `peer` object has getters and setters, I would argue that best practice would be to use them, change to `peer.routerId`